src/mpid/ch4/src/ch4_send.h

```diff
@@ -46,7 +46,7 @@ MPL_STATIC_INLINE_PREFIX int MPID_Send(const void *buf,
 #ifndef MPIDI_CH4_EXCLUSIVE_SHM
```

Comment:
@halimamer what's your plan with this `MPID_THREAD_CS_ENTER(EP, MPIDI_CH4_Global.ep_locks[ep_idx]);`, which requires `ep_idx` to be calculated here?